app/js/checkAnswers.ts

```diff
@@ -130,6 +130,7 @@ function trackActiveQuestion(id: number) {
  * this checks the answers and marks them to show the finishing page
  */
 function showResults(pageLeft) {
+    document.exitFullscreen();
```

Comment:
Can't test if this works. At end of test, when selecting overview, the window came out of fullscreen with a simultaneous alert (as though we had left the screen). On one test this alert may have caused us to fail the test as it seems to technically count coming out of fullscreen as the cursor leaving the window for a moment.